core/pom.xml

```diff
@@ -89,7 +84,6 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <scope>test</scope>
```

Comment:
I thought our goal was no Guava dependency -- are we giving up on that, or is this temporary?  Is there any hope of splitting off a `jdbi-guava` module instead of adding it to core?  (This will probably be answered by reading further, just noting things as I read top-to-bottom)